Update index.js

gabrielsroka opened the pull request:
`Invoke-RestMethod` converts the response from JSON to a PowerShell native object.

From https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/invoke-restmethod
> If the request returns JSON strings, `Invoke-RestMethod` returns a PSObject that represents the strings

why are we calling `ConvertTo-Json` to convert it back to JSON? it's more useful to leave it as is. if you want just the raw text, use `Invoke-WebRequest` instead

abhijitkane commented:
@shreys7 Can you take a look at this?

umeshp7 commented:
@shreys7 any updates on this?

shreys7 commented:
> @shreys7 any updates on this?

We can remove the `ConvertTo-Json` if we wish. As initially it was added to the snippet for tests, now that we have only unit tests and the concept of footer snippet while testing, we can remove this. @umeshp7

shreys7 commented:
@gabrielsroka Thanks for the fix. Although can you just delete the `| ConvertTo-Json` part. As we still want to have the printing of native PSObject code to be included in the generated snippet. Also, see if you can update the unit tests accordingly.

gabrielsroka commented:
@shreys7 
i added back the rest of the code and just removed the `ConvertTo-Json` part.

i'm not sure where the unit test is (and how to update it...). any clues?

shreys7 commented:
@gabrielsroka There are unit tests for checking the snippet generation for PowerShell here:
https://github.com/postmanlabs/postman-code-generators/blob/develop/codegens/powershell-restmethod/test/unit/convert.test.js

It tests for the snippet generated against some hard coded strings. Here, you need to remove the `ConvertTo-JSON` part to correctly test the snippets.

gabrielsroka commented:
see https://github.com/postmanlabs/postman-code-generators/pull/145

(i couldn't figure out how to add the new file to this PR)

umeshp7 commented:
Closing, as new PR created.
